tests/test_logging.py

```diff
@@ -0,0 +1,111 @@
+from datetime import datetime
+import os
+
+
+def test_log():
+    """Test basic log operation outside model operation"""
+    from tm2py.config import LoggingConfig
+    import tempfile
+
+    with tempfile.TemporaryDirectory() as temp_dir:
+        log_config = {
+            "display_level": "STATUS",
+            "run_file_path": "log_run.txt",
+            "run_file_level": "STATUS",
+            "log_file_path": "log.txt",
+            "log_file_level": "DEBUG",
+            "log_on_error_file_path": "error.txt",
+            "notify_slack": False,
+            "use_emme_logbook": False,
+            "iter_component_level": None,
+        }
+
+        class Config:
+            logging = LoggingConfig(**log_config)
+
+        class Controller:
+            config = Config()
+            run_dir = temp_dir
+            iter_component = None
+
+        class TestException(Exception):
+            pass
+
+        controller = Controller()
+
+        from tm2py.logger import Logger
+
+        try:
```

Comment:
avoid try/excepts in tests generally - unless you are testing that an exception if caught

Reply:
See above, testing logging in case of error.